### PR TITLE
feat: update project via cli command

### PR DESCRIPTION
**Pull request by Chinedu19**
# Description

> Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

> Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- OS with version:
- Browser with version:
- SDK / CLI with version:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I documented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/platform/pages/api/cli/v0/projects/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import type { NextApiResponse } from "next";
+import { getProjects } from "@/utils/api/projects";
+import { createProject } from "@/utils/api/projects/create";
+import withCliAuth, { type NextCliApiRequest } from "@/utils/withCliAuth";
+
+const projects = async (req: NextCliApiRequest, res: NextApiResponse) => {
```
Comment (chetannn) [inline apps/platform/pages/api/cli/v0/projects/index.ts:6]: We replaced the `v0` to `v1` so we don't need any files from `v0`.

File: apps/cli/src/commands/project/create.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          auth: {
+            username: cliId as string,
+            password: cliToken as string,
+          },
+        },
+      );
+
+      if (response) {
+        const data = response.data as NewProjectResponse;
+        await loader.stop(
```
[the remark this answers is not in the document]
Comment (Chinedu19) [reply apps/cli/src/commands/project/create.ts]: Fixed, I just discovered I missed this comment

File: apps/cli/src/commands/project/update.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    }),
+  };
+
+  public async run(): Promise<void> {
+    const { flags } = await this.parse(ProjectUpdate);
+
+    let allProjects: ProjectDataType[] = [];
+
+    const config = await getCliConfig();
+    // These are the
```
Comment (dahal) [inline apps/cli/src/commands/project/update.ts:56]: Please remove these comments

**Review by dahal — CHANGES_REQUESTED** [review]
Overall looks good to me. Please add [Project update](https://envless.dev/docs/cli/project#update) documentation as well

https://github.com/envless/envless/blob/main/apps/docs/src/pages/cli/project.mdx#update

**Comment by Chinedu19** [comment]
just did so.

thanks for the review @dahal  and @chetannn

File: apps/platform/env/index.mjs [hunk 4]
```diff
@@ -92,6 +92,7 @@ if (!!process.env.SKIP_ENV_VALIDATION == false) {
   );
 
   if (parsed.success === false) {
+    console.log(parsed, "--parsed");
```
Comment (dahal) [inline apps/platform/env/index.mjs:95]: Please remove this comment

Comment (dahal) [inline apps/platform/env/index.mjs:95]:
```suggestion
```

File: apps/platform/pages/api/cli/v1/projects/update.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import withCliAuth, { NextCliApiRequest } from "@/utils/withCliAuth";
+import prisma from "@/lib/prisma";
+
+type updateRequestData = Partial<{
+  [key: string]: boolean | string | string[];
+}>;
+
+const update = async (req: NextCliApiRequest, res: NextApiResponse) => {
+  if (req.method != "POST") {
+    // Any order request aside the POST request
```
Comment (dahal) [inline apps/platform/pages/api/cli/v1/projects/update.ts:11]:
```suggestion
```

**Pull request merged**